layouts/partials/widgets/tags.html

```diff
@@ -8,11 +8,11 @@ <h3 class="panel-title">{{ i18n "tagsTitle" }}</h3>
     </div>
 
     <div class="panel-body">
-        <ul class="tag-cloud">
+        <ul class="tag-cloud list-inline">
             {{ $current := . }}
             {{ range $name, $items := .Site.Taxonomies.tags }}
-            <li{{ if eq $current.RelPermalink ($name | urlize | lower | printf "/tags/%s/") }} class="active"{{ end }}>
-                <a href="{{ "tags/" | relURL }}{{ $name | urlize | lower }}"><i class="fas fa-tags"></i> {{ $name }}</a>
+            <li class="list-inline-item">
+                <a{{ if eq $current.RelPermalink ($name | urlize | lower | printf "/tags/%s/") }} class="active"{{ end }} href="{{ $.Site.BaseURL }}tags/{{ $name | urlize | lower }}"><i class="fas fa-tags"></i> {{ $name }}</a>
```

Comment:
> There's a known issue this port. Highlighting of active tags does not work. 

I didn't test your PR and only briefly looked at the changes you made, but I think broken highlighting could be due to the fact that you don't assign the class `active` to the `<li>` element anymore (you chose to assign it to the `<a>` element instead). What's your reason to change this?

(same applies to the categories partial `layouts/partials/widgets/categories.html`)

Reply:
I am not a frontend developer, so I am feeling my way through this. However, active category highlighting works. You'll see that once you test the example site.

<img width="1164" alt="screenie" src="https://user-images.githubusercontent.com/4692391/70260970-6b6f2700-175f-11ea-8fcf-5596b76211cf.png">

Reply:
This also won't be the first time I've written code that works and I am not sure why it works.

Reply:
This is how it's handled on the upstream Boostrapious demo site here: https://demo.bootstrapious.com/universal/2-0-2/blog-medium.html

Reply:
I've run your code in Hugo and it turns out you could just add the following CSS to the `static/css/style.*.css` files to enable proper tag highlighting (example for `style.default.css`):

```css
.tag-cloud a.active {
  color: #ffffff;
  background-color: #4fbfa8;
}
```

Reply:
I don’t plan on making this change for this PR. I’ll look at what they do upstream.